app/src/main/res/values-pl-rPL/values.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <integer-array name="notification_vibration_option_values">
+    </integer-array>
+
+    <string-array name="pref_entry_values_appbar_compact_mode">
```

Comment:
And this one I guess is serious confusion by the translating app and should be deleted, please confirm